ci/properties/generator-generic-ossf-slsa3-publish.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "SLSA Generic generator",
+    "creator": "Open Source Security Foundation (OpenSSF)",
+    "description": "Generate SLSA3 provenance for your existing release workflows",
+    "iconName": "go-ossf-slsa3-publish",
+    "categories": ["Continuous integration"]
```

Comment:
If the generator works for multiple languages then it would be good idea to list those languages here. This will make this workflow show up more prominently for repos with those languages.

Reply:
Thank you, great suggestion